web/polymer_elements/paper_toolbar/adding_button_rows/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <link rel="import" href="packages/polymer_elements/paper_toolbar.html">
```

Comment:
Same as above, no Dart code in this example. I assume in this case it's necessary to use html imports. I assumed it isn't worth it to introduce an `index.dart` file just to be able to use `@HtmlImport()`